module/tools/FilterTester/src/FilterTester.cpp

```diff
@@ -0,0 +1,63 @@
+#include "FilterTester.hpp"
+
+#include <filesystem>
+
+#include "extension/Configuration.hpp"
+
+#include "message/input/Sensors.hpp"
+#include "message/platform/RawSensors.hpp"
+
+#include "utility/nbs/Decoder.hpp"
+#include "utility/support/ProgressBar.hpp"
+
+namespace module::tools {
+
+    using extension::Configuration;
+    using message::input::Sensors;
+    using message::platform::RawSensors;
+    using NUClear::message::CommandLineArguments;
+
+    FilterTester::FilterTester(std::unique_ptr<NUClear::Environment> environment)
+        : Reactor(std::move(environment)), config{} {
+
+        on<Configuration>("FilterTester.yaml").then([this](const Configuration& cfg) {
+            // Use configuration here from file FilterTester.yaml
+            this->log_level = cfg["log_level"].as<NUClear::LogLevel>();
+        });
+
+        // This is the information that will come back from the sensor filter
+        on<Trigger<Sensors>, Sync<FilterTester>>().then([this](const Sensors& sensors) {
+            // TODO do something with the sensors coming out of the module
```

Comment:
Do you have any specific plans on what to do with these sensors? Rather than just having "something".

Reply:
I was going to leave that to you guys!